README.md

```diff
@@ -21,9 +25,12 @@ Sketch.App Patch Tool, brought to you by [@duraki](https://github.com/duraki) &
 ## Magic Trick ✨
 For the people who would like to try the new version, we got you covered. You can pass `-m` argument for the ultimate life-saving trick, which will automagically download latest Sketch app from the official website and patch the bundle, ready to be launched from the Applications folder.
 
-And do it in one command: `bash -c "$(curl -s https://raw.githubusercontent.com/duraki/SketchCrapp/master/sketchcrapp.sh -o -)" -O -m`
+[@gordio](https://github.com/gordio) added<br />And do it in one command: `bash -c "$(curl -s https://raw.githubusercontent.com/duraki/SketchCrapp/master/sketchcrapp.sh -o -)" -O -m` 
```

Comment:
I'd remove `[@gordio](https://github.com/gordio) added<br />`. He is already listed as a contributor on the README commits.

![Screenshot 2021-02-09 at 09 47 37](https://user-images.githubusercontent.com/5892610/107338462-dcc10d00-6abb-11eb-9dfb-3c79233885c4.png)

Reply:
Ok